pyproject.toml

```diff
@@ -3,15 +3,15 @@ requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 [project]
 name = "one_py_sdk"
-version = "1.0.1"
+version = "1.0.2"
 dependencies = [
     "requests",
     "protobuf==3.20.1", 
     "one_interfaces",
     "requests_cache"
     ]
 authors = [
-  { name="AquaticInformatics", email="info@aquaticinformatics.com" },
+  { name="AquaticInformatics", email="daniel.edwards@aquaticinformatics.com" },
```

Comment:
Are you sure you want to use your email?

Reply:
Haha